app/pages/Auction/BidActionPanel/BidNow.js

```diff
@@ -105,6 +106,15 @@ class BidNow extends Component {
     }
   }
 
+  processValue = (val, field) => {
+    const value = val.match(/[0-9]*\.?[0-9]{0,6}/g)[0];
+    if (value * consensus.COIN > consensus.MAX_MONEY)
+      return;
+    if (Number.isNaN(parseFloat(value)))
```

Comment:
Wouldn't you want the parseFloat/isNaN check to happen before performing multiplication?

Reply:
You're right, fixing. Although it turns out `NaN * Number = NaN` and `NaN > Number` is `false` so the first check would pass but the second would fail and return...